package.json

```diff
@@ -30,6 +30,7 @@
     "eslint-plugin-astro": "^1.2.3",
     "husky": "^9.0.11",
     "is-ci": "^3.0.1",
+    "playwright": "^1.46.0",
```

Comment:
I think this could be moved in `e2e/package.json`? Or was it a specific reason it was required here?

Reply:
It was needed earlier when `pnpm exec playwright install chromium --with-deps` was run in the root. I've now added `working-directory: e2e` there and it seems to work. Removed this dep from root `package.json`.